tests/unittests/Foundation/NSRecursiveLockTests.m

```diff
@@ -20,143 +20,6 @@
 #import <thread>
 #import <mutex>
 #import <condition_variable>
-#import <chrono>
-
-#import "NSRecursiveLock+Internal.h"
```

Comment:
is there a non private way we can rewrite the test? like an isLocked or something?